packages/webview_flutter/test/webview_flutter_test.dart

```diff
@@ -115,20 +233,45 @@ class FakePlatformWebView {
 
   MethodChannel channel;
 
-  String lastUrlLoaded;
+  LinkedList<_HistoryNode> history = LinkedList<_HistoryNode>();
```

Comment:
As this is only going to be used in the test, and the list is currently small I'd optimize for readability over performance here.
I think using `List` instead of `LinkedList` and an index will result in simpler code.